src/main/java/com/limechain/transaction/TransactionProcessor.java

```diff
@@ -118,12 +121,22 @@ private byte[] processTransaction(Extrinsic extrinsic, PeerId peerId) {
             return transactionState.addToPool(validTransaction);
         }
 
-        return transactionState.shouldAddToQueue(validTransaction)
+        var result = transactionState.shouldAddToQueue(validTransaction)
```

Comment:
`var result` is a bit confusing. Does the method return the extrinsic bytes that were added to the pool/queue or the hex value that we return in the rpc request?

Reply:
TransactionState methods for adding a transaction to pool/queue return blake2b hash of the extrinsic.